### DISCUSSION TITLE
Change time unit for tasks

### DISCUSSION THREAD

**Discussion by xtophe06** [kanboard/kanboard#5408] [Q&A]
Hello,
I’d like to change tasks time unit from "hour(s)" to "day(s)" .
I checked in the settings but did not find how to do this.
Can you please help
Note: I posted also the question on kanboard.discourse.group but did not get any answer.
Thanks
BR
Chris

**Comment by fguillot** [comment]
It's not possible unless you are willing to modify the source code by yourself.

**Comment by xtophe06** [comment]
OK thanks for your answer. I wanted to make sure that this functionnality was not available somewhere. 
Anyway thanks for this tool it's great to work with.
